js/features/status/all.js

```diff
@@ -48,28 +48,40 @@ function extendRejudgePage() {
   });
 }
 
-function _createRadioForm(form, callback) {
+function _createCheckForm(form, callback) {
```

Comment:
사소하지만, 이름은 checkbox 가 어떨까요?

그리고 지금보니까 앞의 `_` 은 큰 의미가 없을 것 같아서 이번 기회에 지워져도 되겠네요 ㅎㅎ

```suggestion
function createCheckboxForm(form, callback) {
```

Reply:
done in [bf1f1c6](https://github.com/joonas-yoon/boj-extended/pull/125/commits/bf1f1c61d1b29ab6f06c782027822043de7d4ef9)